website/docs/docs/cloud/account-settings.md

```diff
@@ -10,6 +10,12 @@ The following sections describe the different **Account settings** available fro
 
 ## Git repository caching <Lifecycle status="enterprise" />
 
+:::important repo caching enabled by default
+
+dbt Cloud Git repository caching is enabled by default for all new Enterprise accounts. To prevent dbt Cloud from caching your repo content, disable the feature prior to integrating with Git.
```

Comment:
I wonder if we should mention why they need to disable prior to integrating with git? Will they need to disconnect and reconnect to Git or something?